packages/checkbox/theme/lumo/vaadin-checkbox-styles.js

```diff
@@ -28,17 +28,17 @@ registerStyles(
     }
 
     :host([has-label]) ::slotted(label) {
-      padding-block: var(--lumo-space-xs);
-      padding-inline: var(--lumo-space-xs) var(--lumo-space-s);
+      padding-block: var(--vaadin-checkbox-label-padding-v, var(--lumo-space-xs));
+      padding-inline: var(--vaadin-checkbox-label-padding-h, var(--lumo-space-xs) var(--lumo-space-s));
```

Comment:
Why separate properties, instead of a single `padding` property?

Reply:
I wasn't sure if `padding` shortcut respects RTL, but apparently it does. Changed to use single property.

Reply:
Thanks for checking. I didn't even consider that aspect.